Constitution.txt

```diff
@@ -1,490 +1,264 @@
 
 Article IV (Article 4 - States' Relations)
 Section 1
-Full Faith and Credit shall be given in each State to the public Acts, Records, and judicial Proceedings of every other State. And the Congress may by general Laws prescribe the Manner in which such Acts, Records and Proceedings shall be proved, and the Effect thereof.
+1: Full Faith and Credit shall be given in each State to the public Acts, Records, and judicial Proceedings of every other State. And the Congress may by general Laws prescribe the Manner in which such Acts, Records and Proceedings shall be proved, and the Effect thereof.
+
+2: The Government of each State shall have the right to challenge the Government of any other state to prove its Acts, Records, and Proceedings, and their compliance with this Constitution and with the Constitution of the state. When such a challenge is issued, the supreme Court shall have original jurisdiction.
 
 Section 2
 1: The Citizens of each State shall be entitled to all Privileges and Immunities of Citizens in the several States.
 
 2: A Person charged in any State with Treason, Felony, or other Crime, who shall flee from Justice, and be found in another State, shall on Demand of the executive Authority of the State from which he fled, be delivered up, to be removed to the State having Jurisdiction of the Crime.
 
-3: No Person held to Service or Labour in one State, under the Laws thereof, escaping into another, shall, in Consequence of any Law or Regulation therein, be discharged from such Service or Labour, but shall be delivered up on Claim of the Party to whom such Service or Labour may be due.11
-
 Section 3
-1: New States may be admitted by the Congress into this Union; but no new State shall be formed or erected within the Jurisdiction of any other State; nor any State be formed by the Junction of two or more States, or Parts of States, without the Consent of the Legislatures of the States concerned as well as of the Congress.
+1: New States may be admitted by the Congress into this Union; but no territory of any State shall be transferred to another State, or be formed into a new State as a member of the Union, unless by three-fourths assent of the people of said territory; and if joining a previously existing State, by the assent of the Government of that State according to rules laid out in the State's Constitution, or if forming a new State, with such a State Constitution as may satisfy the requirements laid upon all States by this Constitution; and no new State shall be formed from the territories of existing States which shall have a population of less than thirty thousand Citizens. When such a State is newly formed, the Citizens of that state shall be treated as having Citizenship of these United States for such term as they have been Citizens of their previous State, or of the newly admitted State.
```

Comment:
Here we adjust the structure of the relationship between Citizen and State.

The State is composed by its Citizens, and exists only at their behest. Therefore, regardless of the will of the Legislature of a State, any group of its Citizens must have the right to leave it and to form their own, or to join another State. I formalize the process here, so that such an action can take place in a clear framework. This will allow Citizens to hold their States to account, not only by their vote for Representatives in their State Legislature, but by their total withdrawal in the event that the State ignores the needs of its people.

Recognizing the need for prudence and care in such a process, since it will have significant political, administrative, and economic impact, I set the bar high for such an action, forcing anyone who wants to do it to find boundaries which will separate two populations that have significant political differences. There is also another element which will slow this dynamic, which I laid out earlier.

Interstate tariffs are one of the few powers I add to Congress in this update. This is one more place where they provide a benefit. Any region which wants to secede from its current state and form a new one will have to recognize the costs that its residents will pay in additional tariffs, by removing themselves politically from areas with which they have significant economic ties. I expect there will still be a substantial number of regions which would act on this process, but they will have to do so accepting those costs.